packages/babel-plugin-i18n-calypso/package.json

```diff
@@ -20,5 +20,8 @@
   },
   "keywords": [],
   "author": "Alex Kirk <alex.kirk@automattic.com> (https://automattic.com)",
-  "license": "GPL-2.0+"
+  "license": "GPL-2.0+",
+  "scripts": {
+    "build": "node ../../bin/build-package"
```

Comment:
Are we okay with relative paths here for now? In the long run, this should invoke a binary from a `calypso-sdk` package -- but we're in a bit of a catch-22 situation here, since we need this PR for #30933 -- which is one of the preparatory steps for such a package!

Reply:
yeah i think this is ok for now.